src/itdelatrisu/opsu/options/OptionsOverlay.java

```diff
@@ -1350,6 +1359,27 @@ private void adjustSlider(int mouseX, int mouseY) {
 		}
 	}
 
+	/**
+	 * Increments or decrements slider value if user presses right or left arrow key.
+	 * @param key the pressed key
+	 */
+	private void adjustSliderWithKey(int key) {
+		int oldSliderValue = hoverOption.getIntegerValue();
+
+		// increment or decrement value
+		if (key == Input.KEY_RIGHT) {
+			hoverOption.setValue(hoverOption.getIntegerValue() + 1);
```

Comment:
You have `oldSliderValue` that holds `hoverOption.getIntegerValue()`; you may as well use it here instead of calling the getter again. Same goes for the line below.